test/cli_labrc/.labrc

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Does the find rc module support file extensions? Being able to specify .js would be nice at least for syntax highlighting.